lib/storage/adapters/embedded.js

```diff
@@ -119,6 +124,8 @@ EmbeddedStorageAdapter.prototype._put = function(key, item, callback) {
 
   item.shard = null; // NB: Don't store any shard data here
 
+  item.fskey = crypto.createHash('sha1').update(key).digest('hex');
```

Comment:
For the purpose of keeping consistent with the hash function used elsewhere in storj-lib ([and the recent discovery that sha1 is dead](http://shattered.io/)), let's replace this with:

```
utils.rmd160(key, 'hex');
```